src/components/_carousel.scss

```diff
@@ -62,6 +62,7 @@
 
     img {
         width: 100%;
+        height: auto;
```

Comment:
Not too sure how this is helping, could you demonstrate with a video the before and after, please?